common/output-model.cpp

```diff
@@ -1124,13 +1124,13 @@ void frame_drops_dashboard::draw(ux_window& win, rect r)
     ImGui::Text("%s", "Measurement Metric:"); ImGui::SameLine();
     ImGui::SetCursorPosY(ImGui::GetCursorPosY() - 3);
 
-    ImGui::SetCursorPosX(200);
+    ImGui::SetCursorPosX( 11.5f * win.get_font_size() );
 
     std::vector<const char*> methods;
     methods.push_back("Viewer Processing Rate");
     methods.push_back("Camera Timestamp Rate");
 
-    ImGui::PushItemWidth(r.w - 207);
+    ImGui::PushItemWidth(-1.f);
```

Comment:
What does a width of -1 do?
Looks like the older calculation is related to windows size, don't we need it?

Reply:
-1 value set width "to the end available zone"
`ImGui::Text("SetNextItemWidth/PushItemWidth(-1)");
 ImGui::SameLine(); HelpMarker("Align to right edge");
 ImGui::PushItemWidth(-1);
` 
The code sample from imgui_demo.cpp [here](https://skia.googlesource.com/external/github.com/ocornut/imgui/+/refs/tags/v1.78/imgui_demo.cpp#2109)

Reply:
Can we do the same with all combo boxes in the left pane and adjust the border according to the font size?